platform/security/certificate/security-certificate-keystoreeditor/src/main/java/org/codice/ddf/security/certificate/keystore/editor/KeystoreEditor.java

```diff
@@ -330,6 +328,7 @@ public void addTrustedCertificate(
           e);
     } finally {
       IOUtils.closeQuietly(socket);
+      IOUtils.closeQuietly(fos);
```

Comment:
❓ Do we really want to close this output stream quietly? It would mean the file was not properly created.

Reply:
It will catch any IO errors above:

`fos = Files.newOutputStream(trustStoreFile);
 trustStore.store(fos, keyStorePassword.toCharArray());
 } catch (IOException | GeneralSecurityException e) {
 LOGGER.info(
          "Unable to add certificate(s) to trust store from URL: {}",
          (decodedUrl != null) ? decodedUrl : url,
          e);
}`

The close quietly is just to avoid a null fos causing issues.